### PR TITLE
reword the hints when the user cannot write to a chat

### PR DESCRIPTION
**Pull request by r10s**
tackles https://github.com/deltachat/deltachat-desktop/issues/1225

just a suggestion. feel free to change the text or to close this or :)

### CODE REVIEW

File: _locales/_untranslated_en.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
     "message": "Copy link"
   },
   "messaging_disabled_not_in_group": {
-    "message": "Messaging disabled (you are not in this group)"
+    "message": "You are not in this group and cannot write therefore. To join, ask another member."
   },
   "messaging_disabled_device_chat": {
-    "message": "Messaging disabled (this is the device chat)"
+    "message": "This chat contains locally generated messages and can be read only."
```
Comment (hpk42) [inline _locales/_untranslated_en.json]:
```suggestion
    "message": "This chat contains notification messages generated by the Delta Chat application running on your device."
```

maybe?

Comment (r10s) [reply _locales/_untranslated_en.json]: i think the point is to clarify that the user cannot _type_ - the text appears where normally the input field takes place. otoh, it should not look like an error or as if the user has made sth. wrong.

also, there is already the subtitle that says "Locally generated messages" and the device message that explains further.

Comment (hpk42) [reply _locales/_untranslated_en.json]: k -- in any case i think the phrase "can be read only" is not proper english i think :)

Comment (hpk42) [inline _locales/_untranslated_en.json]:
```suggestion
    "message": "This chat contains locally generated messages. Therefore you can not write any messages."
```

**Pull request merged**